pkg/cmd/pulumi/templates/org.go

```diff
@@ -62,6 +62,9 @@ func (s *Source) getOrgTemplates(
 		}
 		logging.Infof("could not get a backend for org templates")
 		return
+	} else if b == nil {
```

Comment:
Looks like everywhere else already handles nil, nil result from Current. Maybe worth updating the doc comment on it thought that nil, nil is a valid return value.